pom.xml

```diff
@@ -334,13 +337,23 @@
             <dependency>
                 <groupId>org.springframework.amqp</groupId>
                 <artifactId>spring-rabbit</artifactId>
-                <version>1.2.2.RELEASE</version>
+                <version>${spring-amq.version}</version>
```

Comment:
Is this property used in more than one place?  If not, I suggest removing it.  In my opinion properties are for when we want to use the same value in multiple places.  I understand that hasn't been consistent throughout projects.

Reply:
I saw the use of a property here as a way to see all of the versions in a concise way.  I can see advantages for both styles.  I'll remove the property for and we can come to a consensus some other time.